src/openrct2/rct12/RCT12.h

```diff
@@ -320,54 +320,54 @@ namespace OpenRCT2::RCT12
     };
 } // namespace OpenRCT2::RCT12
 
-enum
-{
-    MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT = (1 << 7),
-};
-
-constexpr uint8_t RCT2_STRING_FORMAT_ARG_START = 123;
-constexpr uint8_t RCT2_STRING_FORMAT_ARG_END = 141;
-constexpr uint8_t RCT2_STRING_FORMAT_COLOUR_START = 142;
-constexpr uint8_t RCT2_STRING_FORMAT_COLOUR_END = 156;
+constexpr uint8_t kRCT12StringFormatArgStart = 123;
+constexpr uint8_t kRCT12StringFormatArgEnd = 141;
+constexpr uint8_t kRCT12StringFormatColourStart = 142;
+constexpr uint8_t kRCT12StringFormatColourEnd = 156;
 
-constexpr uint8_t RCT12_SOUND_ID_NULL = 0xFF;
+constexpr uint8_t kRCT12SoundIdNull = 0xFF;
 
 using RCT12RideId = uint8_t;
-constexpr RCT12RideId RCT12_RIDE_ID_NULL = 255;
+constexpr RCT12RideId kRCT12RideIdNull = 255;
 
-constexpr uint8_t RCT12_BANNER_INDEX_NULL = std::numeric_limits<uint8_t>::max();
+constexpr uint8_t kRCT12BannerIndexNull = std::numeric_limits<uint8_t>::max();
 
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_EDGE_STYLE_MASK = 0xE0;   // in RCT12TileElement.properties.surface.slope
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_WATER_HEIGHT_MASK = 0x1F; // in RCT12TileElement.properties.surface.terrain
-constexpr uint8_t RCT12_TILE_ELEMENT_SURFACE_TERRAIN_MASK = 0xE0;      // in RCT12TileElement.properties.surface.terrain
+constexpr uint8_t kRCT12SurfaceElementEdgeStyleMask = 0xE0;         // in RCT12TileElement.properties.surface.slope
+constexpr uint8_t kRCT12SurfaceElementWaterHeightMask = 0x1F;       // in RCT12TileElement.properties.surface.terrain
+constexpr uint8_t kRCT12SurfaceElementTerrainMask = 0xE0;           // in RCT12TileElement.properties.surface.terrain
+constexpr uint8_t kRCT12SurfaceElementTypeSurfaceMask = 0b00000011; // in RCT12TileElement.properties.surface.type
+constexpr uint8_t kRCT12SurfaceElementTypeEdgeMask = 0b01000000;    // in RCT12TileElement.properties.surface.type
 
-constexpr uint8_t RCT12_SMALL_SCENERY_ELEMENT_NEEDS_SUPPORTS_FLAG = 0x20;
-constexpr uint8_t RCT12_TILE_ELEMENT_COLOUR_MASK = 0b0001'1111;
+constexpr uint8_t kRCT12SmallSceneryElementNeedsSupportsFlag = 0x20;
+constexpr uint8_t kRCT12TileElementColourMask = 0b0001'1111;
 
-constexpr uint16_t RCT12_TILE_ELEMENT_LARGE_TYPE_MASK = 0x3FF;
+constexpr uint16_t kRCT12TileElementLargeTypeMask = 0x3FF;
 
-constexpr uint16_t const RCT12_XY8_UNDEFINED = 0xFFFF;
+constexpr uint8_t kRCT12TrackElementTypeFlagChainLift = 1 << 7;
+constexpr uint8_t kRCT12TrackElementSequenceGreenLight = 1 << 7;
+
+constexpr uint16_t const kRCT12xy8Undefined = 0xFFFF;
```

Comment:
I think this should be `kRCT12XY8Undefined`. Where does the 8 come in, btw?

Reply:
The actual struct is called `RCT12xy8`. The 8 is the number of bits for each coordinate (RCTxy16 also exists.)